system/jlib/jflz.cpp

```diff
@@ -665,6 +670,7 @@ class jlib_decl CFastLZCompressor : public CInterface, public ICompressor
         outlen = 0;
         outbuf = NULL;      // only set on close
         bufalloc = 0;
+        wrmax = 0x10000;
```

Comment:
Is there any point significance in setting to 0x10000?
As invalid / can't be used unless open() called, which sets..
Might be better to set to 0 and add comment that set by open.